assets/texts/fr/amoDescription.html

```diff
@@ -32,33 +32,42 @@
 <ul>
 	<li>Affichage des emojis dans le style d'Apple, Google, Twitter, EmojiOne ou tout simplement ce que votre navigateur utilise.</li>
 	<li>Recherche rapide d'un émoticône avec différents mots-clés.</li>
-	<li>Search for emojis <a href="https://github.com/rugk/awesome-emoji-picker/wiki/FAQ#how-to-use-the-address-bar-to-search-for-emojis">directly in the address bar</a>.</li>
-	<li>Supports the dark/black mode of your system automatically.</li>
+	<li>Recherchez des émojis <a href="https://github.com/rugk/awesome-emoji-picker/wiki/FAQ#how-to-use-the-address-bar-to-search-for-emojis">directement dans la barre d'adresse</a>.</li>
+	<li>Prend en charge le thème sombre de votre système automatiquement.</li>
 	<li>Copie des émoticônes dans le presse-papier ...</li>
 	<li>... ou insertion directe dans la page courante.</li>
 	<li>Mémorisation des émoticônes utilisés fréquemment.</li>
 	<li>Prise en compte et mémorisation du teint des émoticônes.</li>
 	<li>Possible insertion des syntaxes à <code>:deux-points:</code> des émojis au lieu de l'émoji en Unicode utilisé par défaut.</li>
-	<li>Utilisation du raccourcis-clavier (par défaut <code>Ctrl</code>+<code>Maj</code>+<code>:</code>) pour accèder au sélecteur.</li>
-	<li>Intégration dans Firefox Sync, pour avoir les même paramètrages sur vos appareils.</li>
-	<li>Ajustement de l'apparence et de nombreux autres paramètrages selon vos goûts.</li>
+	<li>Utilisation du raccourcis-clavier (par défaut <code>Ctrl</code>+<code>Maj</code>+<code>:</code>) pour accéder au sélecteur.</li>
+	<li>Intégration dans Firefox Sync, pour avoir les même paramétrages sur vos appareils.</li>
+	<li>Ajustement de l'apparence et de nombreux autres paramétrages selon vos goûts.</li>
 	<li>La taille du sélecteur et des émojis est également ajustable.</li>
-	<li>Utilisation d'un icône coloré ou monochrome pour la barre d'outils.</li>
+	<li>Utilisation d'un icône colorée ou monochrome pour la barre d'outils.</li>
```

Comment:
It's correct to change "coloré" to "colorée" but then also change "un icône" to "une icône"

Reply:
Jeez, that's a dumb mistake 😞

Thank you for pointing it out